src/main/scala/cromwell/util/TryUtil.scala

```diff
@@ -6,11 +6,12 @@ import scala.util.{Failure, Try}
 
 object TryUtil {
   def stringifyFailures[T](failure: Try[T]): String = {
-    val writer = new PrintWriter(new StringWriter())
+    val stringWriter = new StringWriter()
```

Comment:
In the case of `StringWriter` that's probably true, but not a bad habit to adopt... :smile: